Cubical/Experiments/LocalisationDefs.agda

```diff
@@ -1,146 +0,0 @@
--- This file contains several ways to define localisation
```

Comment:
@mzeuner  Should this be kept or deleted?

Reply:
I don't think there's any need to keep this file around.

Reply:
I'll delete it.